### DISCUSSION TITLE
Kickoff meeting time and location

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#21] [Polls]
We should have a "live" kickoff meeting so people can chat and meet. Should this be a social event, or a work event? Where should it be held?

One option is in ~5 weeks at EGU. Another is Zooms (multiple, both due to time zones and general availability). It seems likely there will be an EGU meet-up even if we have a Zoom meeting before then.

A `work event` means that we would use this meeting to make decisions about what/who/how to do next steps. 

A `social event` means just chat casually and informally, but not work or make any formal decisions because it's hard to include everyone in one meeting and hard to coordinate decisions across multiple meetings.

**Comment by fmaussion** [comment]
I voted for a work zoom - @mankoff and I will work out the agenda based on suggestions from discussions here. I suggest two zooms with same agenda to accommodate for time zones.

**Reply by mankoff** [reply]
Scheduling poll at https://www.when2meet.com/?24069882-U5hEM

**Comment by gunterl** [comment]
These are all good suggestions. I voted for a work zoom as well. I like meeting in person at a conference only when there is a good effort to have most of the people attending it (and possibly helping those with travel assistance if needed). And I find meeting in person to be very valuable as opposed to zoom. Getting to know others at a personal level is also super great and I would be in favor in meeting with others socially at EGU if anyone is interested.

**Comment by oien2** [comment]
I am looking forward to the Zoom meeting when it's scheduled. I am also attending EGU in person this year and it would be great to socially meet people there too!
